### ISSUE TITLE
Themes are not working except 'min' and 'nord'

### ISSUE THREAD

**Issue by soaibsafi** [EldoranDev/gridsome-plugin-remark-shiki#19]
I was trying to ` 'material-theme-darker'` from the [Shiki](https://github.com/shikijs/shiki/tree/master/packages/themes).
```
transformers: {
    remark: {
      externalLinksTarget: '_blank',
      externalLinksRel: ['nofollow', 'noopener', 'noreferrer'],
      plugins: [
        ['gridsome-plugin-remark-shiki', {
          theme: 'material-theme-darker', skipInline: true
        }]
      ]
    }
  },
```

I also tried other themes. No other themes are working except `min` and `nord`. 
I am getting this error- 
![Capture](https://user-images.githubusercontent.com/11424812/94362124-6e098700-00b9-11eb-9dff-8256711e36e6.PNG)

**Comment by EldoranDev** [comment]
Hi, do you have an repository to look at for that error ?

**Comment by soaibsafi** [comment]
Hi @EldoranDev,
I was trying to use this template-
https://github.com/cossssmin/gridsome-starter-bleda

**Comment by EldoranDev** [comment]
Should be fixed with v0.4.0
